ci/homebrew-tap/ci-deploy.sh

```diff
@@ -29,4 +30,10 @@ if [[ $do_push == true ]]; then
   git push origin master
 else
   echo "Refraining from push since it's a dry run"
+  sed -i "s/pypi.io/$pypi_host/" git-machete.rb
+  # install git-machete from local formula with homebrew
+  /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" < /dev/null
+  echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"' >> /home/circleci/.bash_profile
+  eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+  brew install ./git-machete.rb
 fi
```

Comment:
Also! pls note that this pollutes the current CI VM... would be useful to check if that interferes e.g. with `git-machete` installed via snap

Reply:
i'll remove it with `brew remove git-machete` after version check.